.gitignore

```diff
@@ -1,2 +1,4 @@
 /*.out
 /*.test
+/.idea
```

Comment:
Use .git/info/exclude for this, it's not a project artifact, it's an artifact of your build environment. https://stackoverflow.com/q/22906851